manifests/tekton/tasks/base/cosa-upload-containers.yaml

```
@@ -0,0 +1,35 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cosa-upload-containers
+spec:
+  steps:
+    - image: 'quay.io/coreos-assembler/coreos-assembler:latest'
+      name: container-build-and-push
+      resources:
+        limits:
+          devices.kubevirt.io/kvm: '1'
+          memory: 4Gi
+        requests:
+          devices.kubevirt.io/kvm: '1'
+          memory: 4Gi
+      script: |
+        #!/usr/bin/env bash
+        set -euxo pipefail
+
+        cd /workspace/coreos
+        cosa build-extensions-container
+        cosa push-container --image=extensions-container  --authfile=/tmp/dockerconfig.json  quay.io/okd/centos-stream coreos-9:devel
```

For now, we can push just the base container, and buid and push the extensions later once https://github.com/openshift/os/pull/991 has merged.
IIRC the image tag that is pushed to should be generated automatically from the build version
```suggestion
        #cosa build-extensions-container
        cosa push-container quay.io/okd/centos-stream-coreos-9
        #cosa push-container --image=extensions-container quay.io/okd/centos-stream-coreos-9-extensions
```

---

OK, kept only push-container, base container